psydac/linalg/__init__.py

```diff
@@ -1,4 +1,4 @@
-__all__ = ['basic', 'block', 'direct_solvers', 'iterative_solvers', 'stencil', 'kron', 'utilities']
+__all__ = ['basic', 'block', 'direct_solvers', 'iterative_solvers', 'stencil', 'kron', 'topetsc', 'utilities']
 
```

Comment:
I think that `__all__` is not useful in this `__init__` module (which we probably want to remove in the future, anyway). Further it is not even up to date, as it is missing the `identity` entry.

Hence please remove `__all__` from this module.

Reply:
done in [68937d4](https://github.com/pyccel/psydac/pull/244/commits/68937d4ffbcceec954f6f2e2bef2287f88f62853)